src/backend/aspen/api/settings.py

```diff
@@ -120,6 +121,7 @@ class Settings(BaseSettings):
 
     # Env vars usually pulled from AWS SSM Parameters
     AWS_NEXTSTRAIN_SFN_PARAMETERS: Dict
+    AWS_PANGOLIN_SFN_PARAMETERS: Dict
```

Comment:
FastAPI didn't start pangolin jobs before, so it didn't need a config option for it.